src/main/resources/com/google/api/codegen/nodejs/index.snip

```diff
@@ -0,0 +1,43 @@
+@extends "nodejs/common.snip"
+
+@snippet generate(index)
+  /*
+   {@copyright()}
+   */
+  'use strict';
+
+  @join require : index.requireViews
+    var {@require.clientName} = require('./{@require.fileName}');
+  @end
+  var gax = require('google-gax');
+  var extend = require('extend');
+  @if index.hasMultipleServices
+    var union = require('lodash.union');
+  @end
+
+  function {@index.apiVersion}(options) {
+    options = extend({
+      scopes: {@index.apiVersion}.ALL_SCOPES
+    }, options);
+    var gaxGrpc = gax.grpc(options);
+    var result = {};
+    @join require : index.requireViews
+      extend(result, {@require.clientName}(gaxGrpc));
+    @end
+    return result;
```

Comment:
It would be more idiomatic here to return the client directly if there is only one.

ie:
```
@switch index.requireViews.size.toString
@case "1"
  return {@require.clientName}(gaxGrpc);
@default
  var result = {};
  @join require : index.requireViews
    extend(result, {@require.clientName}(gaxGrpc));
  @end
  return result;
@end
```